.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         # The Windows build currently fail because of https://github.com/golang/go/issues/40795, and because xcaddy isn't compatible with the known workaround
         #os: [ ubuntu-latest, macos-latest, windows-latest ]
         os: [ ubuntu-latest, macos-latest ]
-        go: [ '1.19' ]
+        go: [ '1.20' ]
```

Comment:
Can bump CI to 1.21 even. We just released 2.7.4 with support for that.

go.mod should stay 1.20 for now though